README.md

```diff
@@ -37,14 +45,22 @@ def info_dump(customers)
 end
 ```
 
+
 Snippet 3 - Big O:
+O(1) CONSTANT
+# ... no matter how many elements are in this array, this algorithm is only looking at one element in each array.
+
 ```ruby
 def first_element_is_red?(array)
   array[0] == 'red' ? true : false
 end
 ```
 
+
 Snippet 4 - Big O:
+O(n^2) EXPONENTIAL
+# ... b/c these are two linear loops nested within one another, so n*n is exponential. CRYSTAL, can we go over what this loop is doing in class? I looked up next if--and the loop below does not seem to do what I'd expect.
+
```

Comment:
```
hess in ~/Documents/github-repos/textbook-curriculum (master)
$ pry
[1] pry(main)> 10.times do |i|
[1] pry(main)*   next if i == 5
[1] pry(main)*   puts i
[1] pry(main)* end  
0
1
2
3
4
6
7
8
9
=> 10
```

`next` is saying, don't do anything more in this loop iteration if the follow condition is true (cause there's a condition connected to it; `next` could be standalone.)

So basically, in this case, it's saying, if I'm comparing an element with itself (index1 == index2, since both loops are iterating over the same array), then don't return true if those values match (item1 == item2) because.. they should match--they're the same index/element!